pyramid_nacl_session/session.py

```diff
@@ -1,22 +1,26 @@
+# -*- coding: utf-8 -*-
+
 from pyramid.session import (
     BaseCookieSessionFactory,
     PickleSerializer,
 )
 
 from .serializer import EncryptedSerializer
 
+
+# pylint: disable=too-many-arguments,invalid-name
 def EncryptedCookieSessionFactory(
-    secret,
-    cookie_name='session',
-    max_age=None,
-    path='/',
-    domain=None,
-    secure=False,
-    httponly=False,
-    timeout=1200,
-    reissue_time=0,
-    set_on_exception=True,
-    serializer=None,
+        secret,
+        cookie_name='session',
+        max_age=None,
+        path='/',
+        domain=None,
+        secure=False,
+        httponly=False,
+        timeout=1200,
+        reissue_time=0,
+        set_on_exception=True,
+        serializer=None,
 ):
```

Comment:
Side note, if you use [Black](https://github.com/psf/black) for code formatting, I would not be opposed. We use it in Pyramid now.

Reply:
:heavy_check_mark: